### ISSUE TITLE
Crash when trying to combine non-botania cosmetics

### ISSUE THREAD

**Issue by yrsegal** [williewillus/Botania#239]
If either the ICosmeticAttachable or the ICosmeticBauble aren't ItemBauble, crash. ¯\_(ツ)_/¯ 

The weird part is, it's a rendering crash.

```
Caused by: java.lang.NullPointerException
    at net.minecraft.client.renderer.RenderItem.renderItemOverlayIntoGUI(RenderItem.java:425) ~[RenderItem.class:?]
    at net.minecraft.client.gui.inventory.GuiContainer.drawSlot(GuiContainer.java:319) ~[GuiContainer.class:?]
    at net.minecraft.client.gui.inventory.GuiContainer.drawScreen(GuiContainer.java:118) ~[GuiContainer.class:?]
    at net.minecraft.client.renderer.InventoryEffectRenderer.drawScreen(InventoryEffectRenderer.java:62) ~[InventoryEffectRenderer.class:?]
    at net.minecraft.client.gui.inventory.GuiInventory.drawScreen(GuiInventory.java:82) ~[GuiInventory.class:?]
    at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:313) ~[ForgeHooksClient.class:?]
    at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1147) ~[EntityRenderer.class:?]
```

**Comment by yrsegal** [comment]
to reproduce, build https://github.com/yrsegal/NaturalPledge from source.

**Comment by williewillus** [comment]
Full log?

**Comment by yrsegal** [comment]
I'm not able to get a log right now, i'll get it back to you when I can

**Comment by yrsegal** [comment]
But that should be sufficient for solving this..?

**Comment by williewillus** [comment]
i don't know where in botania code that crash is originating

**Comment by yrsegal** [comment]
that's the thing. It WASN'T. However, this was the only recipe that caused it.

**Comment by williewillus** [comment]
I have no idea where this is coming from, if you could track it down in your mod that'd be great

**Comment by yrsegal** [comment]
I don't know, either. There's nothing about Natural Pledge or Botania in the crashlog, yet it only happens with them.

**Comment by yrsegal** [comment]
Do you have a `uwotm8` label like on upstream? Because this deserves a `uwotm8`.

**Comment by yrsegal** [comment]
resolved in 1.9.4, somehow?

**Comment by williewillus** [comment]
lmao
